### PR TITLE
module 1: Hello microverse project

### PR DESCRIPTION
**Pull request by haroldmud**
## Hello world project setup
We added : 
- we configured HTML, CSS, and Github actions lints
- We added HTML, CSS and README files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
...
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- node.js & npm
+
+### Setup
+
+- git clone \<project name>
```
Comment (Deepakdanger) [inline README.md:30]: - [ ] **Required** Let's add here the actual code to clone this repo.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,75 @@
...
+
+<!-- ### Deployment -->
+
+## Authors
+
+👤 **Harold**
+
+- GitHub: [@haroldmud](https://hargithub.com/haroldmud)
+- Twitter: [@mudosaharold](https://twitter.com/twitterhandle)
+- LinkedIn: [@haroldmudosa](https://linkedin.com/in/linkedinhandle)
```
Comment (Deepakdanger) [inline README.md:49]: - [ ] **Required** Lets add the correct link of author's `github` `twitter` & `LinkedIn`.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,75 @@
...
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- node.js & npm
+
+### Setup
+
+- git clone \<project name>
+- cd \<project name>
```
Comment (Deepakdanger) [inline README.md:31]: - [ ] **Required**  Let's add here the actual ` project name` to clone this repo.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @haroldmud, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 
- Good PR ✔️ 
- No Linter error ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] While committing message ,please consider
- All commits should be present tense.
- The first letter should start with capital letters. For ex:- `Add linter files`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
Hi @haroldmud , 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**